apps/pie-docs/test/system/navigation.spec.js

```diff
@@ -1,33 +1,38 @@
+import { test, expect } from '@playwright/test';
+import { disableCookieBanner } from '../helpers/playwright-helper';
 import expectedRoutesJson from '../snapshots/expected-routes.snapshot.json';
 
-describe('PIE - Status Code Tests', async () => {
-    expectedRoutesJson.forEach((route) => {
-        it(`Should respond with a '200' status code for route - ${route}`, async () => {
-            const puppeteer = await browser.getPuppeteer();
-            const [page] = await puppeteer.pages();
-            const url = `${browser.options.baseUrl}/${route}`;
+test.beforeEach(async ({ page, context }) => {
+    await page.goto(process.env.BASE_URL);
+    await disableCookieBanner(page, context);
+});
 
+test.describe('PIE - Status Code Tests - @desktop', () => {
+    expectedRoutesJson.forEach((route) => {
+        test(`Should respond with a '200' status code for route - ${route}`, async ({ page }) => {
+            // Arrange
+            const url = `${await page.url()}/${route}`;
             const response = await page.goto(url);
 
+            // Assert
             await expect(response.status()).toBe(200);
         });
     });
+});
 
-    it('Should open and close the mobile navigation menu', async () => {
-        await browser.url(browser.options.baseUrl);
-
-        await browser.emulateDevice('iPhone X');
-
-        const navToggleLabelSelector = '[data-test-id="nav_toggle_label"]';
-        const navToggleInputSelector = '[data-test-id="nav_toggle_input"]';
-
-        const navToggleLabel = await browser.$(navToggleLabelSelector);
-        const navToggleInput = await browser.$(navToggleInputSelector);
+test.describe('PIE - site nav menu - @mobile', () => {
+    test('Should open and close the mobile navigation menu', async ({ page }) => {
+        // Arrange
+        const navToggleLabel = page.getByTestId('nav_toggle_label');
+        const navMenu = page.getByTestId('site_nav');
 
+        // Act - Open nav menu
         await navToggleLabel.click();
-        await browser.waitUntil(async () => navToggleInput.isSelected());
+
+        // Act - Close nav menu
         await navToggleLabel.click();
 
-        await expect(navToggleInput).not.toBeSelected();
+        // Assert
+        await expect(navMenu).not.toBeVisible();
```

Comment:
it might have never opened. I think we need to asset its visible after the first click